www/src/pages/getting-started/introduction.mdx

```diff
@@ -5,6 +5,7 @@ import CssCodeBlock from '../../components/CssCodeBlock';
 import DocLink from '../../components/DocLink';
 import ReactPlayground from '../../components/ReactPlayground';
 import AsPropButtonExample from '../../examples/AsProp';
+import AsPropAccordionExample from '../../examples/AsPropComponent';
```

Comment:
The import name should be `AsPropComponent`

```
import AsPropComponent from '../../examples/AsPropComponent';
```

Reply:
Oh! I apologise, @kyletsang; I changed the import name.